### DISCUSSION TITLE
How to use it in android for videos bult in apps

### DISCUSSION THREAD

**Discussion by mspbypmp** [aclap-dev/video-downloadhelper#259] [Help and Q&A]
### Your question

How to use it in android  for videos streamed through applications where videos are played only while using the app.What I mean to say that one has to use the app and videos can be accessed through particular app only with no download option.

### Addon & CoApp version and details

How to use it in android  for videos streamed through applications where videos are played only while using the app.What I mean to say that one has to use the app and videos can be accessed through particular app only with no download option.

**Comment by paulrouget** [comment]
Sorry I don't understand what you're describing.

VDH works on browsers. Does the browser plays a videos streamed from android?

**Reply by mspbypmp** [reply]
it works on browser on a pc.
is there an android version of video downloadhelper that can download videos inbuilt in learning applications? not in browser.

**Reply by GrampaWildWilly** [reply]
So you're talking about a cell phone.  You have an app on your phone, not part of the browser, that you use for taking education courses.  And you want VDH to download the course material from that app, without going into the browser.  Do I have that right?

The question of an Android version of VDH/CoApp came up in the old Google forum a few years ago.  Michel's answer at the time was that there were no plans to build such a thing.  But VDH is a browser extension.  Even if there were an Android version of VDH/CoApp, it would be in the browser app on the cell phone.  I don't see why you would expect VDH to work with some app that isn't part of the browser.

Unless I've completely misunderstood you.

**Reply by paulrouget** [reply]
Right, Video DownloadHelper doesn't work on Android.

**Reply by mspbypmp** [reply]
Now somebody got my question.I know it works on browser.I was just enquiring whether theres a separate ability built for android for capturing inbuilt videos in applications.
 **"I don't see why you would expect VDH to work with some app that isn't part of the browser".**
 The answer to this is bcoz i expect vdh to download every protected content where there is a limitation.I found the application as fantastic as I downloaded contenet 2 days back where none of the other softwares worked.Thats y asked.

**Reply by GrampaWildWilly** [reply]
Are you saying you have installed VDH on your cell phone & it works in your browser?  How did you do that?  A tutorial would be most welcome, complete with screenshots off your phone.  Do tell us which browser you are using on your cell phone.  If you could give us some screenshots off your phone of VDH downloading something on an Android web browser, that would probably be of interest.  Have you rooted your phone?  What operating system are you running on your phone?  Plain old Android or something custom you got from somewhere?  What phone are you using?  I'm asking for brand & model of phone.  I don't mean to pry but I think this is a relevant question.  What country do you live in?  It is my understanding that the phone may be a certain model of a certain brand, but what I would have available to me in the US is not the same as what is available in another country, even though on the surface it may look like the same phone.  Here, it seems like all the phone manufacturers & service providers have colluded to prevent rooting.

As for VDH working with some random app, that isn't true on desktop Mac, Linux, or Windows.  You should not expect that on Android.  VDH doesn't work in, for example, Notepad++ or Open Office on my Windows 11 desktop.  It works in Firefox, nowhere else.  It does not seem to me like a logical request for VDH to work anywhere outside any browser, neither on a desktop nor on a cell phone.
